src/test/java/au/id/deejay/webserver/server/WebServerTest.java

```diff
@@ -19,7 +20,8 @@
 /**
  * @author David Jessup
  */
-@RunWith(MockitoJUnitRunner.class)
+//@RunWith(MockitoJUnitRunner.class)
```

Comment:
I would think you're still going to need Mockito. These tests may not run, otherwise. Am I wrong?

Also, please delete instead of comment.

Reply:
On second thought, maybe we should treat Mockito like JUnit and also replace it with JMH.